pkg/trace/api/otlp.go

```diff
@@ -520,8 +525,17 @@ func (o *OTLPReceiver) convertSpan(rattr map[string]string, lib pcommon.Instrume
 	for k, v := range rattr {
 		setMetaOTLP(span, k, v)
 	}
+
+	spanKind := spanKindName(in.Kind())
+	if o.conf.OTLPReceiver.ComputeTopLevelBySpanKind && (spanKind == "server" || spanKind == "consumer") {
```

Comment:
Are there constants for these span kinds somewhere? (not super important, but if these are used in multiple places it might be nice)